### ISSUE TITLE
Course website needs to be generalized to accommodate new workshops

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/reproducible-research#103]
Ideally we'd do this before the planned 2022 October session, but if we don't have the bandwidth, it would be sufficient to update the schedule, etc.

**Comment by Jen-OMalley** [comment]
Similar to [**this comment**](https://github.com/AlexsLemonade/training-admin/issues/743#issuecomment-1222389164), I wanted to confirm how we want to share details about the course in upcoming email outreach. Do we want to share the [**current RRP website**](https://alexslemonade.github.io/reproducible-research/) as an example? (Would probably want to at least remove the logistics section to avoid confusion.) Or do we just describe the content of the course for now?  Including the entire `What you will learn` and `What you won't learn` sections will probably make the email too lengthy but maybe we can take some of the [**blog post**](https://www.ccdatalab.org/blog/data-lab-reproducibility-workshop-philadelphia-area-june-10-2022) language.

Tagging **@jaclyn-taroni @jashapiro @sjspielman** for opinions! 
cc: **@allyhawkins @cbethell**

**Comment by jaclyn-taroni** [comment]
* If you approve #106, we can include a link to the website in email communications. 
* I'd include an abbreviated version of what you will learn, what you won't learn in sentence form.

_Process comment:_ I think email communications could have used its own issue (possibly two) set to blocking https://github.com/AlexsLemonade/training-admin/issues/741, which is apparent to me mostly because I've come back from a day off, and the discussion of emails are not where I would expect.

**Comment by Jen-OMalley** [comment]
Makes sense! I would consider email communications "in progress" still, so I can file a separate issue.

**Comment by Jen-OMalley** [comment]
Hm should we remove the June schedule from the website for now as well? Or include a note that this is an example from a past workshop? (I can file the PR to do either!)

**Comment by jaclyn-taroni** [comment]
Note sounds good. I left it in there because it links to materials, which may help people get a better sense of what they are signing up for!

**Comment by sjspielman** [comment]
I'm going to take this out of the YI epic because we're using a different website over at https://github.com/AlexsLemonade/2022-yi-workshops/ for both Intro R and RRP at YI 2022.
